homeassistant/components/ituran/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "domain": "ituran",
+  "name": "Ituran",
+  "codeowners": ["@shmuelzon"],
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/ituran",
+  "integration_type": "hub",
+  "iot_class": "cloud_polling",
+  "requirements": ["pyituran==0.1.2"]
```

Comment:
Your library itself is missing a LICENSE file. You do declare it in setup.py, but it doesn't hurt to be more explicit with licensing

Reply:
Added a license file to the library and bumped its version in HA.